app/models/king.rb

```diff
@@ -1,2 +1,19 @@
 class King < ChessPiece
+  X_MOVE_FACTOR = 1
+  Y_MOVE_FACTOR = 1
+
+  def valid_move?(destination_x, destination_y)
+    fail ArgumentError, 'Invalid Move' if obstructed?(destination_x, destination_y)
+    exceeds_x_movement?(destination_x) || exceeds_y_movement?(destination_y) ? false : true
```

Comment:
Russian Literature and 10 years of reading and writing legal docs...lol...love it :+1: